32.mender-convert/docs.md

```diff
@@ -7,6 +7,413 @@ shortcode-core:
 github: false
 ---
 
+## mender-convert 2.6.1
+
+_Released 01.24.2022_
+
+### Statistics
+
+A total of 15314 lines added, 8824 removed (delta 6490)
+
+| Developers with the most changesets | |
+|---|---|
+| Lluis Campos | 94 (21.1%) |
+| Ole Petter Orhagen | 77 (17.3%) |
+| Kristian Amlie | 67 (15.1%) |
+| Mirza Krak | 55 (12.4%) |
+| Drew Moseley | 37 (8.3%) |
+| Adam Podogrocki | 35 (7.9%) |
+| Dominik Adamski | 18 (4.0%) |
+| Eystein Måløy Stenberg | 17 (3.8%) |
+| Marek Belisko | 8 (1.8%) |
+| Fabio Tranchitella | 4 (0.9%) |
+
+| Developers with the most changed lines | |
+|---|---|
+| Mirza Krak | 6745 (34.3%) |
+| Ole Petter Orhagen | 4274 (21.7%) |
+| apodogrocki | 2597 (13.2%) |
+| Adam Podogrocki | 2026 (10.3%) |
+| Lluis Campos | 1325 (6.7%) |
+| Kristian Amlie | 842 (4.3%) |
+| Eystein Måløy Stenberg | 518 (2.6%) |
+| Drew Moseley | 334 (1.7%) |
+| Dell Green | 327 (1.7%) |
+| Dominik Adamski | 241 (1.2%) |
+
+| Developers with the most lines removed | |
+|---|---|
+| Mirza Krak | 1463 (16.6%) |
+| Simon Guigui | 1 (0.0%) |
+
+| Developers with the most signoffs (total 9) | |
+|---|---|
+| Lluis Campos | 3 (33.3%) |
+| Adam Podogrocki | 3 (33.3%) |
+| Kristian Amlie | 2 (22.2%) |
+| Yevgeniy Nurseitov | 1 (11.1%) |
+
+| Top changeset contributors by employer | |
+|---|---|
+| Northern.tech | 357 (80.2%) |
+| RnDity | 57 (12.8%) |
+| open-nandra | 8 (1.8%) |
+| Griesser AG | 3 (0.7%) |
+| Ideaworks Ltd | 3 (0.7%) |
+| bakern@gmail.com | 3 (0.7%) |
+| github@survive.ch | 3 (0.7%) |
+| Lafon Technologies | 1 (0.2%) |
+| TideWise Ltda. | 1 (0.2%) |
+| Cisco Systems, Inc. | 1 (0.2%) |
+
+| Top lines changed by employer | |
+|---|---|
+| Northern.tech | 14256 (72.5%) |
+| RnDity | 4864 (24.7%) |
+| Ideaworks Ltd | 327 (1.7%) |
+| open-nandra | 117 (0.6%) |
+| bakern@gmail.com | 30 (0.2%) |
+| Griesser AG | 15 (0.1%) |
+| fyhertz@gmail.com | 13 (0.1%) |
+| Cisco Systems, Inc. | 11 (0.1%) |
+| INNOBYTE | 11 (0.1%) |
+| github@survive.ch | 7 (0.0%) |
+
+| Employers with the most signoffs (total 9) | |
+|---|---|
+| Northern.tech | 5 (55.6%) |
+| RnDity | 3 (33.3%) |
+| enurseitov@gmail.com | 1 (11.1%) |
+
+| Employers with the most hackers (total 30) | |
+|---|---|
+| Northern.tech | 11 (36.7%) |
+| RnDity | 3 (10.0%) |
+| Ideaworks Ltd | 1 (3.3%) |
+| open-nandra | 1 (3.3%) |
+| bakern@gmail.com | 1 (3.3%) |
+| Griesser AG | 1 (3.3%) |
+| fyhertz@gmail.com | 1 (3.3%) |
+| Cisco Systems, Inc. | 1 (3.3%) |
+| INNOBYTE | 1 (3.3%) |
+| github@survive.ch | 1 (3.3%) |
+
+### Changelogs
+
+#### mender-convert (2.6.1)
```

Comment:
This is a huge changelog for being a bugfix release :thinking:

Reply:
I will simply run this manually, and see. Hopefully update, and then we can handle this at a later point.